tests/google.rs

```diff
@@ -56,7 +56,8 @@ async fn fetch_google() {
     // Send off the request by first negotiating an SSL handshake, then writing
     // of our request, then flushing, then finally read off the response.
     let connector = async_native_tls::TlsConnector::new();
-    let mut socket = t!(connector.connect("google.com", socket).await);
+    let url = url::Url::parse("https://google.com/").unwrap();
```

Comment:
why this change? is it actually required?

Reply:
Ah, I added this for testing that things worked with passing an Url struct. It's not required. If you want I remove it.